src/passes/Inlining.cpp

```diff
@@ -199,11 +235,55 @@ struct Inlining : public Pass {
     // return whether we did any work
     return inlined.size() > 0;
   }
+
+  bool canInline(int numUses) {
+    return numUses == SINGLE_USE;
+  }
+
+  bool worthInlining(Function* func) {
+    return Measurer::measure(func->body) <= INLINING_SIZE_LIMIT;
```

Comment:
Oh, one other question: if we are restricting inlining to single-use non-address-taken non-exported functions, is it ever *not* worthwhile to inline? It seems like removing a function (along with its declaration, locals, maybe signature, etc) would always result in a smaller binary (with the caveat that we might not want the functions to be too huge. Should we increase the size limit as long as we maintain these other restrictions?